tests/lean/test_extern.lean.expected.out

```diff
@@ -1,5 +1,5 @@
-test_extern.lean:7:0-7:17: error: test_extern: HAdd.hAdd does not have an @[extern] attribute
-test_extern.lean:9:0-9:86: error: native implementation did not agree with reference implementation!
+test_extern.lean:6:0-6:17: error: test_extern: HAdd.hAdd does not have an @[extern] attribute or @[implemented_by] attribute
+test_extern.lean:8:0-8:86: error: native implementation did not agree with reference implementation!
 Compare the outputs of:
```

Comment:
Why does the error message say to compare the outputs instead of showing what the outputs are? I think it should actually print the outputs (when possible), because these issues can be OS dependent so it's important to know what value was actually produced in CI since they may not reproduce locally.

Reply:
That's a good suggestion, but I'd prefer not to take it on now. Happy to either open an issue, let you do that. Or, for that matter, to accept commits on this PR, or promise to review a new PR! :-)